buildSrc/buildSrc/src/main/kotlin/Deps.kt

```diff
@@ -0,0 +1,30 @@
+object AndroidSdk {
+	const val min = 24
+	const val compile = 30
+	const val target = compile
+}
+
+object BuildPlugins {
+	const val androidGradle = "com.android.tools.build:gradle:${Versions.androidGradle}"
+	const val kotlinGradlePlugin = "org.jetbrains.kotlin:kotlin-gradle-plugin:${Versions.kotlin}"
+}
+
+object Libs {
+	const val kotlinStd = "org.jetbrains.kotlin:kotlin-stdlib:${Versions.kotlin}"
+	object Android {
+		const val appCompat = "androidx.appcompat:appcompat:${Versions.appCompat}"
+		const val constraintLayout = "androidx.constraintlayout:constraintlayout:${Versions.constraintlayout}"
+	}
+
+	//Testing
+	const val mockk = "io.mockk:mockk-android:${Versions.mockk}"
```

Comment:
would be possible to setup jUnit5 for unit test instead of `mockk`? 

reference: https://github.com/radityagumay/COVID-19-ANDROID/commit/9a08321d7f1c71d156e5ee4937393acc4f891aa9

Reply:
I think they can be used together.
https://mockk.io/#junit5
I don't sure about how to set them up through.
Let remove it for now, then we can have another issue/pr to merge with some sample test later.

Reply:
cool mas, thank you